.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,49 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if command is '/request_review'
+        id: check-command
+        run: |
+          if [[ $(echo "${{ github.event.comment.body }}" | tr '[:upper:]' '[:lower:]') == "/request_review"* ]]; then
+            echo "::set-output name=command::true"
+          else
+            echo "::set-output name=command::false"
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Add label and request review
+        if: steps.check-command.outputs.command == 'true'
+        run: |
+          reviewers=$(echo "${{ github.event.comment.body }}" | grep -oE "@[a-zA-Z0-9_\-]+" | grep -v "@cryostatio/reviewer(s)" | paste -sd "," -)
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          if [ -n "$reviewers" ]; then
+            echo "Requesting review from reviewers: $reviewers"
+            gh pr request-review ${{ github.event.issue.number }} --reviewer $reviewers
+          else
+            echo "Requesting review from @cryostatio/reviewer(s) team"
+            gh pr request-review ${{ github.event.issue.number }} --team cryostatio/reviewer(s)
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Remove label on changes requested
```

Comment:
I think this step should be run on a review event:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review

We might need to separate this into a another workflow file.

Reply:
Also, I think it does not have to be `REQUEST_CHANGES` state. As long as there is a review, the label can be removed.

Reply:
sometimes the review is just a comment and nothing needs to be changed. Or as long as someone added their review it should have satisfy the condition of review requested since someone already reviewed it? is that what you mean? it makes sense still.

Reply:
> I think this step should be run on a review event:
> 
> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review
> 
> We might need to separate this into a another workflow file.

I think it makes more sense to add this step here to remove the label when review is added. @andrewazores what do u think?

Reply:
I think removing the label when any review is added makes sense. It means somebody has indeed reviewed the PR, even if they just ended up leaving a comment or asking questions without requesting particular changes or approving it, so I imagine the next step in the review process is just some discussion, and then maybe an approval or maybe some more code changes come. Either way, the label doesn't need to be there anymore. If that workflow doesn't end up making sense in practice we can always come back and change this easily enough.